site/docs/transform/timeunit.md

```diff
@@ -22,7 +22,9 @@ Vega-Lite supports the following time units:
 
 Vega-Lite time units can also be a string of consecutive time units to indicate desired intervals of time. For example, `yearmonthdate` indicates chronological time sensitive to year, month, and date (but not to hours, minutes, or seconds). The specifier `monthdate` is sensitive to month and date, but not year, which can be useful for binning time values to look at seasonal patterns only.
 
-By default, all time units represent date time using local time. To use UTC time, you can add the `utc` prefix (e.g., `"utcyear"`, `"utcyearmonth"`).
+- By default, all time units represent date time using local time.
+- To use UTC time, you can add the `utc` prefix (e.g., `"utcyear"`, `"utcyearmonth"`).
+- For timeUnit in encoding, you can also add `"binned"` prefix (e.g., `"binnedyearmonth"` or `"binnedutcyearmonth"`) for Chronological time unit (i.e., units that are truncated date time)
```

Comment:
It's another semantic-based way to explain date trunc (which focus on action being done on the date object, rather than the semantics).

It's a term that Jeff uses in Vega doc. https://vega.github.io/vega/docs/transforms/timeunit/ 

I added a bit more explanation so I hope it makes more sense now.